index.html

```diff
@@ -67,7 +72,7 @@ <h3>Via a <code>&lt;textarea></code></h3>
 
 		<h3>Via a <code>&lt;pre>&lt;code></code></h3>
 
-		<p>If the primary use case is displaying code, it makes sense to initialize from a <code>&lt;pre>&lt;code></code>, same as the one you'd write for using Prism.
+		<p>If the primary use case is displaying code, it makes sense to initialize from a <code>&lt;pre>&lt;code></code>, same as the one you"d write for using Prism.
```

Comment:
What happened here?

Reply:
Find and replace mistake 😨 . This is fixed now.